### ISSUE TITLE
Cannot draw chart for narrow range values

### ISSUE THREAD

**Issue by Cito** [pyrochlore/obsidian-tracker#4]
The example Weight Log chart looks good because it has yMin = 60 and yMax = 80, and only that range of values is shown, not the full range starting from yMin = 0. However, if the range of values is only between yMin = 65 and yMax = 80, suddenly the chart shows up with yMin = 0, even if there is no value smaller than 65 and even if you set yMin = 65 explicitly.

This is because of the code used [here](https://github.com/pyrochlore/obsidian-tracker/blob/59eb43bc20b171e1348452a82c1af8ed889b42ef/src/graph.ts#L240):
```js
    if ((yMin >= 0 && yMin > yMax * 0.8) || (yMin >= 0 && graphInfo.accum)) {
        yScale.domain([0, yMax * 1.2]).range([height, 0]);
    }
```

Since 65 > 80 * 0.8, it forcefully sets the minimum of the yScale to 0. This is not what one would expect, not sure why you are doing this?

**Comment by pyrochlore** [comment]
I did that because in some cases I want to see the zero level of y value for a whole picture.
Since we have a "yMin" parameter to set the minimum y value, the code you mentioned could be removed.
Thanks to mention that. Will make changes in the next release.

**Comment by pyrochlore** [comment]
Fixed and released in version 1.1.0.

**Comment by Cito** [comment]
Thank you. Works nicely now.
